pkg/controllers/disruption/metrics.go

```diff
@@ -75,7 +75,17 @@ var (
 			Name:      "consolidation_timeouts_total",
 			Help:      "Number of times the Consolidation algorithm has reached a timeout. Labeled by consolidation type.",
 		},
-		[]string{consolidationTypeLabel},
+		[]string{ConsolidationTypeLabel},
+	)
+	InvalidatedConsolidationTotal = opmetrics.NewPrometheusCounter(
+		crmetrics.Registry,
+		prometheus.CounterOpts{
+			Namespace: metrics.Namespace,
+			Subsystem: voluntaryDisruptionSubsystem,
+			Name:      "failed_validations_total",
+			Help:      "Number of times a consolidation decision was invalidated. Labeled by consolidation type and reason.",
+		},
+		[]string{ConsolidationTypeLabel, metrics.ReasonLabel},
```

Comment:
We discussed offline and decided to drop the reason label for now since it was tough to reason about what each reason would mean out of context